CPP_STYLE_GUIDE.md

```diff
@@ -0,0 +1,138 @@
+# C++ style guide
```

Comment:
This is capitalized in CONTRIBUTING.md, I'd suggest to do the same here.

Reply:
done!